mountpoint-s3-client/tests/get_object.rs

```diff
@@ -101,3 +106,20 @@ async fn test_get_object_404_bucket() {
         Err(ObjectClientError::ServiceError(GetObjectError::NoSuchBucket))
     ));
 }
+
```

Comment:
It is not correct. I pushed it to get review for the direction to go for testing.

Reply:
Make sure to fix the bucket and keys because this test is looking at a non-existent bucket and key right now.